lightwood/api/json_ai.py

```diff
@@ -1265,6 +1274,10 @@ def adjust(self, train_data: Union[EncodedDs, ConcatedEncodedDs, pd.DataFrame],
     @timed
     def predict(self, data: pd.DataFrame, args: Dict = {{}}) -> pd.DataFrame:
 {predict_body}
+
+    @timed
+    def test(self, data: pd.DataFrame, metrics: List[Callable]) -> pd.DataFrame:
```

Comment:
Add `Callable` in the imports section (see `IMPORTS` variable) near the start of the file.